_drafts/2021-05-04-cultivating-human-connection.md

```diff
@@ -0,0 +1,35 @@
+
+Instructors might endeavor to “humanize” their course content and communications with students. Including an “About Me” page on Canvas helps students gain a sense of their instructor as a person, not just an authority figure who creates and grades assignments. Adding a brief personal component to class-wide announcements-- like sharing something interesting that happened over the weekend, some feelings about current events, or simply wishing students well--also helps students connect with their instructors on a human level. When students see their instructors as more than academic overseers, they are more likely to seek support from them should support be needed.
+
+With a more open, personable online presence, instructors create a transparent learning environment where students are welcome to be just as open and honest with their instructors. In addition to this communication between students and professors being personable, it is crucial that it is clear and prompt as well. Instructors taking action and not only identifying, but supporting struggling individuals is also a means of creating a stronger bond between themselves and their students. Making the first move to check in with those who may be having difficulty with a course, or a particular assignment, can be all that it takes for a student to feel seen and heard-- especially those who may be reluctant to ask for help when they truly need it. If possible, having some sort of weekly check-in (perhaps in the form of a discussion board or email) could be the push some students may need to stay engaged, motivated, and in touch. 
+
+## What Students Can Do
+
+In order to reinvigorate human connection in a digital learning setting, students and instructors need to work together. Instructors are facing a similar increase in workload and stress as students: we are all members of the same world, after all. There are a few things students can keep in mind in order to treat and respect their instructors as fellow human beings.
+
+It seems that because of the authoritative role, students have a tendency to forget that faculty members are people facing the same hard times as everyone else. In digital communications through Canvas or email, students should try to be mindful of their words and tone. Addressing faculty by their proper title and name when composing an email is more respectful than an empty greeting that fails to address that instructor’s humanity, like beginning a message with just “Hi,” or “Hello.” Including a brief statement at the beginning of the message, like “I hope you’re doing well,” shows that the student cares about their professor as a person, not just as an instructor. These are both simple elements students can add to their communications that humanize the student-instructor relationship. The [CDL micro-workshop]*add link* on crafting the perfect extension request email to a professor reiterates these elements and is a good resource for students to refer to.
```

Comment:
This is such a great point, but feels a little redundant here after the previous paragraph. Maybe you can move this sentence up before your sentence that begins with "There are a few things students..."?